src/components/Modals/OnboardingTransfer.vue

```diff
@@ -466,35 +455,6 @@ export default Vue.extend({
       displayOptions.placeholderColor = lightPlaceholderGray;
       this.$emit('updateTransfer', this.displayOptions.exam, this.displayOptions.class, this.tookSwimTest);
     },
-    // Clear a major if a new college is selected and the major is not in it
-    clearSubjectAndScoreIfNotInCollege() {
```

Comment:
This function is unused. Not sure why it is introduced and not sure what it actually does...